sdk/storage/Azure.Storage.Common/tests/Shared/TestConstants.cs

```diff
@@ -19,6 +19,10 @@ public class TestConstants
         public const string Https = "https";
         public const int HttpPort = 443;
 
+        // service cache timeout is 60s, but adding a buffer to be safe
+        public const int DataLakeRetryDelay = 70000;
```

Comment:
Nit - rename to DataLakeCacheInvalidationRetryDelay or similar to make it clear that other folks shouldn't use this particular constant.